code/game/machinery/computer/atmos_control.dm

```diff
@@ -478,45 +446,43 @@ var/global/list/atmos_controllers = list()
 			if(current.rcon_setting == RCON_NO)
 				return 1
 			var/device_id = href_list["id_tag"]
-			switch(href_list["command"])
-				if( "power",
-					"adjust_external_pressure",
-					"set_external_pressure",
-					"set_internal_pressure",
-					"checks",
-					"co2_scrub",
-					"tox_scrub",
-					"n2o_scrub",
-					"o2_scrub",
-					"n2_scrub",
-					"panic_siphon",
-					"scrubbing")
-					var/val
-					if(href_list["val"])
-						val=text2num(href_list["val"])
-					else
-						var/newval = input("Enter new value") as num|null
-						if(isnull(newval))
+			var/command = href_list["command"]
+			if(command in XGM.gases)
+				var/val=text2num(href_list["val"])
+				current.send_signal(device_id, list(command+"_scrub" = val ))
+			else
+				switch(href_list["command"])
+					if( "power",
+						"set_external_pressure",
+						"set_internal_pressure",
+						"checks",
+						"panic_siphon",
+						"scrubbing",
+						"direction")
+						var/val
+						if(href_list["val"])
+							val=text2num(href_list["val"])
+						else
+							var/newval = input("Enter new value") as num|null
+							if(isnull(newval))
+								return 1
+							if(href_list["command"]=="set_external_pressure")
+								if(newval>1000+ONE_ATMOSPHERE)
+									newval = 1000+ONE_ATMOSPHERE
+								if(newval<0)
+									newval = 0
```

Comment:
I know you also didn't write this one, and I'm going to recommend `clamp()` again anyway